rialto_airflow/harvest/doi_sunet.py

```diff
@@ -89,21 +89,23 @@ def get_author_maps(authors):
     df["orcidid"] = df["orcidid"].apply(orcid_id)
 
     # orcid -> sunet
-    orcid = pd.Series(df["sunetid"].values, index=df["orcidid"]).to_dict()
+    orcid: dict[str, str] = pd.Series(
```

Comment:
the `pd.Series()` results were the parts where the type checker asked for a type to be specified.  the rest of the added type specifications in this file just seemed to make things clearer at a glance to me, and passed the type checker.